efficientdet/g3doc/faq.md

```diff
@@ -0,0 +1,30 @@
+# EfficientDet FQA
+
+go/g3doc-canonical-go-links
+
+<!--*
+# Document freshness: For more information, see go/fresh-source.
+freshness: { owner: 'tanmingxing' reviewed: '2020-05-09' }
+*-->
+
+
+[TOC]
+
+## For Users
+
+### How can I convert the saved model to tflite?
+
+Unfortunately, there is no way to do that with the current public tensorflow release due to some issues in tf converter.
+We have some internal fixes, which could potentially be available with the next TensorFlow release.
+
+## For Developers
+
+### How can I format my code for PRs?
+
+    !pylint --rcfile=../.pylintrc your_file.py
```

Comment:
Maybe we could also all agree to use an automatic code formatter in addition to a linter to make style issues just go away. [YAPF](https://github.com/google/yapf) with the `--google` option might be right for this project. I aIso have good experience with the [black](https://github.com/psf/black) formatter. What do you think @mingxingtan ?

Reply:
Excellent idea, and thanks for pointing this out. YAPF is exactly what we need. I updated the FAQ.